docs/Scores - Student Registration Steps.postman_collection.json

```diff
@@ -0,0 +1,231 @@
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{base_url}}/contacts/{{contactId}}",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"contacts",
+						"{{contactId}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Step 3. Update Contact",
+			"request": {
+				"method": "PATCH",
+				"header": [
+					{
+						"key": "client_id",
+						"value": "{{sandbox_client_id}}",
+						"type": "text"
+					},
+					{
+						"key": "client_secret",
+						"value": "{{sandbox_client_secret}}",
+						"type": "text"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"Allergies\": \"List some allergies / other needs\"\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{base_url}}/contacts/{{contactId}}",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"contacts",
+						"{{contactId}}"
+					]
+				}
+			},
+			"response": []
+		},
+		{
+			"name": "Step 4. Finish Registration",
+			"request": {
+				"method": "POST",
+				"header": [
+					{
+						"key": "client_id",
+						"value": "{{sandbox_client_id}}"
+					},
+					{
+						"key": "client_secret",
+						"value": "{{sandbox_client_secret}}"
+					}
+				],
+				"body": {
+					"mode": "raw",
+					"raw": "{\n    \"datetime\": \"2024-06-27T15:45:30Z\",\n    \"waiverResponse\": \"Acceptance\",\n    \"waiverId\": \"a3WU8000000DiRXMA0\"\n}",
+					"options": {
+						"raw": {
+							"language": "json"
+						}
+					}
+				},
+				"url": {
+					"raw": "{{base_url}}/contacts/{{contactId}}/finishRegistration",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"contacts",
+						"{{contactId}}",
+						"finishRegistration"
+					],
+					"query": [
+						{
+							"key": "phoneNumber",
+							"value": "2",
+							"disabled": true
+						}
+					]
+				},
+				"description": "BODY/JSON:\n\n{  \n\"TeamSeasonName\": \"\",  \n\"TeamId\": \"\",  \n\"SeasonId\": \"\",  \n\"SchoolSite\": \"\",  \n\"Partnership\": \"\",  \n\"TotalNoOfPlayers\": 0,  \n\"TotalNoOfSessions\": 0,  \n\"SeasonStartDate\": \"\",  \n\"SeasonEndDate\": \"\",  \n\"CoachSoccer\": \"\",  \n\"CoachWriting\": \"\",  \n\"ProgramCoordinator\": \"\",  \n\"ProgramManager\": \"\"  \n}"
+			},
+			"response": []
+		},
+		{
+			"name": "/contacts/{contactId}",
+			"request": {
+				"method": "GET",
+				"header": [
+					{
+						"key": "client_id",
+						"value": "{{sandbox_client_id}}"
+					},
+					{
+						"key": "client_secret",
+						"value": "{{sandbox_client_secret}}"
+					}
+				],
+				"url": {
+					"raw": "{{base_url}}/contacts/003U8000008Mz3RIAS",
+					"host": [
+						"{{base_url}}"
+					],
+					"path": [
+						"contacts",
+						"003U8000008Mz3RIAS"
```

Comment:
What is this ID?

Reply:
An example to retrieve a contact. It was created while I was testing.